apps/web/pages/order/success.vue

```diff
@@ -1,16 +1,17 @@
 <template>
-  <div class="px-4 md:px-0" data-testid="order-success-page">
+  <div class="px-4 md:px-0" data-testid="order-success-page" v-if="orderData.order">
     <div
       class="border border-1 border-neutral-200 mt-10 mb-20 rounded p-4 md:p-6 flex flex-col items-center max-w-2xl mx-auto"
     >
       <NuxtImg src="/images/order-success.svg" :alt="$t('orderSuccessfulImageAlt')" width="192" height="192" />
       <h1 class="mt-6 mb-1 text-2xl">{{ $t('successInfoHeader') }}</h1>
       <span class="font-medium">{{ $t('successInfoMessage') }}</span>
+      <span v-if="orderData.order.deliveryAddress.options.length > 0" class="font-medium">{{
+        $t('orderConfirmation.confirmationSendTo', { email: orderData.order.deliveryAddress.options[0].value })
```

Comment:
Fusch am Bau, check the option Id to get the email address should be typeId: 5
<img width="870" alt="Bildschirmfoto 2023-08-21 um 16 42 15" src="https://github.com/plentymarkets/storefront-nuxt3-boilerplate/assets/30629157/ca5fb620-fbf7-4230-896a-a2d007d8b559">